cachingController.go

```diff
@@ -52,6 +52,18 @@ func (c *healthCheckController) updateCachedHealth(services map[string]service,
 			newMService := newMeasuredService(service)
 			c.measuredServices[service.name] = newMService
 			refreshPeriod := findShortestPeriod(categories)
+			categories, err := c.healthCheckService.getCategories()
+			if err != nil {
+				warnLogger.Printf("Cannot read categories: [%v]\n Using minimum refresh period for service [%s]\n", err, service.name)
```

Comment:
Do we really need the `\n`s in the log?

Reply:
We don't need the last one, I'll remove it.